app/javascript/find/application.js

```diff
@@ -34,9 +34,9 @@ const filterToggleButton = new FilterToggleButton({
   startHidden: false,
   toggleButton: {
     container: $('.app-filter-toggle'),
-    showText: 'Show filters',
+    showText: 'Filter results',
```

Comment:
Changing here will change the live site and the new results page (until is proper live). Is it expected?